### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by LocalHeroPro**

### CODE REVIEW

File: src/content/1.7/basics/installation/_index.md [hunk 1]
```diff
@@ -87,27 +87,37 @@ This step is only needed if you downloaded the development version.
...
+- ./log
+- ./mails
+- ./modules
+- ./themes
+- ./translations
+- ./var
+
+You can setup proper permissions with that one line:
+```bash
+$ sudo chmod +w -R admin-dev/autoupgrade app/logs app/Resources/translations cache config/themes download img log mails modules themes translations var
```
Comment (matks) [inline src/content/1.7/basics/installation/_index.md]: Hi @LocalHeroPro thanks for the suggestion however this only works on specific environment. Probably most of us work on such an environment, but we cannot exclude those of us who work on specific distros or event ... Windows 😭

Comment (matks) [reply src/content/1.7/basics/installation/_index.md]: Also it's maybe not the best practice to use `sudo` 😉

Comment (LocalHeroPro) [reply src/content/1.7/basics/installation/_index.md]: So maybe prepare those steps and add EXTRA heading, that is for Linux environment? After that everyone - developers - can more friendly reproduce steps for their own environment.

If no `sudo` so what is better?

Comment (PierreRambaud) [reply src/content/1.7/basics/installation/_index.md]: I'm working on Linux and never had to run this kind of command to make my apache / Nginx works with PrestaShop :thinking: 
FYI: if you have a fresh install of Debian, sudo is not installed.

Comment (LocalHeroPro) [reply src/content/1.7/basics/installation/_index.md]: So, I have to remove `sudo` and everything is fine for you gays?

Comment (eternoendless) [reply src/content/1.7/basics/installation/_index.md]: I think it would be good without sudo :)

**Comment by eternoendless** [comment]
Please consider activating the option "Allow edits from maintainers" on your PR.

You can find it at the bottom of the right column:

image

Thanks!

**Comment by LocalHeroPro** [comment]
@eternoendless as I can see, I have already checked that option.

File: src/content/1.7/basics/installation/_index.md [hunk 2]
```diff
@@ -87,27 +87,39 @@ This step is only needed if you downloaded the development version.
...
+- ./mails
+- ./modules
+- ./themes
+- ./translations
+- ./upload
+- ./var
+
+You can setup proper permissions with that one line:
+```bash
+$ chmod +w -R admin-dev/autoupgrade app/logs app/Resources/translations cache config/themes download img log mails modules themes translations var
```
Comment (eternoendless) [inline src/content/1.7/basics/installation/_index.md]:
```suggestion
$ chmod +w -R admin-dev/autoupgrade app/config app/logs app/Resources/translations cache config download img log mails modules themes translations upload var
```

File: src/content/1.7/basics/installation/_index.md [hunk 3]
```diff
@@ -87,27 +87,39 @@ This step is only needed if you downloaded the development version.
...
+- ./img
+- ./log
+- ./mails
+- ./modules
+- ./themes
+- ./translations
+- ./upload
+- ./var
+
+You can setup proper permissions with that one line:
```
Comment (eternoendless) [inline src/content/1.7/basics/installation/_index.md]:
```suggestion
You can set up the appropriate permissions using this command:
```

File: src/content/1.7/basics/installation/_index.md [hunk 4]
```diff
@@ -87,27 +87,39 @@ This step is only needed if you downloaded the development version.
...
+```bash
+$ chmod +w -R admin-dev/autoupgrade app/logs app/Resources/translations cache config/themes download img log mails modules themes translations var
+```
+
+If you do not have any of above folder, please create it before changing permissions. For example:
```
Comment (eternoendless) [inline src/content/1.7/basics/installation/_index.md]:
```suggestion
If you do not have some of the folders above, please create them before changing permissions. For example:
```

**Comment by matks** [comment]
Thank you @LocalHeroPro

**Pull request merged**